src/legacy/core_plugins/kibana/public/dashboard/index.ts

```diff
@@ -37,13 +31,8 @@ import './dashboard_config';
 async function getAngularDependencies(): Promise<LegacyAngularInjectedDependencies> {
   const injector = await legacyChrome.dangerouslyGetActiveInjector();
 
-  const Private = injector.get<IPrivate>('Private');
-
-  const savedObjectRegistry = Private(SavedObjectRegistryProvider);
-
   return {
     dashboardConfig: injector.get('dashboardConfig'),
-    savedObjectRegistry,
     savedDashboards: injector.get('savedDashboards'),
```

Comment:
It looks like we also don't need to inject saved dashboards anymore because we can create the class inside the app using the helper (might be done in a separate PR)

Reply:
thx, I've taken care of this, planned to do it in a 2020 PR, but the time is now!
did it the same way with visualize, so the injected variables are just used for `savedObjectManagementRegistry`. now the first 2020 PR will be cleanup of this 🔨